src/wacksbywarby/db.py

```diff
@@ -74,7 +74,7 @@ def write_timestamp(self, timestamp: datetime):
     def get_timestamp(self) -> Optional[str]:
         try:
             with open(self.timestamp_path) as f:
-                t = f.read()
+                t = f.read().strip()
```

Comment:
could this ever throw when calling .strip()?

Reply:
it looks like `f.read()` always returns a string, so this should be ok